src/data-tutorials/users.tsx

```diff
@@ -1563,6 +1563,13 @@ const Users: User[] = [
     website: 'blog/tutorials/potential-growth-of-blockchain-and-its-use-cases',
     tags: ['celosage', 'beginner'],
   },
+  {
+    title: 'Overview of NFTs on the Celo Platform',
+    description: 'In this tutorial, we explore NFTs and what NFTs platforms are on Celo',
+    preview: require('./showcase/beginner/overview-of-NFTs-on-the-Celo-platform.png'),
```

Comment:
Image path has issues kindly fix it.

Reply:
require('./showcase/beginner/overview-of-NFTs-on-the-Celo-Platform.png')

It should be this.